pkg/bmc/virtualmedia_validation.go

```diff
@@ -0,0 +1,56 @@
+package bmc
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stmcginnis/gofish"
+	"github.com/stmcginnis/gofish/redfish"
+)
+
+// ValidateVirtualMedia returns nil when the BMC has support
+// for VirtualMedia and error when the BMC does not have support.
+func ValidateVirtualMedia(address string, user string, password string, insecure bool) error {
+	bmcConfig := gofish.ClientConfig{
+		Endpoint: address,
+		Username: user,
+		Password: password,
+		Insecure: insecure,
+	}
+
+	client, err := gofish.Connect(bmcConfig)
+	if err != nil {
+		return errors.Wrap(err, "could not connect to redfish BMC.")
```

Comment:
Didn't you want a specific error type, so that you can use it in Installer?

Reply:
:+1: This is a good point. We need to differentiate. We might be able to unwrap and see the underlying HTTP error coming out of the HTTP library, but it'd probably be better to have something here.

Reply:
I've created a new Error in the errors.go